ipinfo/cmd_n2ip.go

```diff
@@ -0,0 +1,50 @@
+package main
+
+import (
+	"fmt"
+	"github.com/ipinfo/cli/lib"
+	"github.com/ipinfo/cli/lib/complete"
+	"github.com/ipinfo/cli/lib/complete/predict"
+	"github.com/spf13/pflag"
+)
+
+// cmdN2IP is the handler for the "n2ip" command.
+var completionsN2IP = &complete.Command{
+	Flags: map[string]complete.Predictor{
+		"--nocolor": predict.Nothing,
+		"-h":        predict.Nothing,
+		"--help":    predict.Nothing,
+		"-6":        predict.Set(predictReadFmts),
+		"--ipv6":    predict.Set(predictReadFmts),
+	},
+}
+
+// printHelpN2IP prints the help message for the "n2ip" command.
+func printHelpN2IP() {
+	fmt.Printf(
+		`Usage: %s n2ip [<opts>] <expr>
```

Comment:
Expr? I didn't think this would do that but we likely wanna let that be a "magic" feature that gets discovered by the user when reading the description. We could show this at this level though so the most common use case is apparent:

```suggestion
		`Usage: %s n2ip [<opts>] <number>
```